client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -354,7 +354,7 @@ const Settings = ( {
 				title={ translate( 'Redirect Domain', { textOnly: true } ) }
 				subtitle={ translate( 'Redirect from your domain to another' ) }
 			>
-				<DomainRedirectCard domainName={ selectedDomainName } nameservers={ nameservers } />
+				<DomainRedirectCard domainName={ selectedDomainName } domain={ domain } />
```

Comment:
Lets drop domainName if we're requiring the full domain object anyway.

```suggestion
				<DomainRedirectCard domain={ domain } />
```

We can access domain name through `domain.name` as needed in the component.

Reply:
Thanks, I rebased and updated your suggestions!